webcompat/issues.py

```diff
@@ -32,6 +32,7 @@ def filter_new(issues):
 
     "new" means anything that isn't an issue with a "contactready",
     "sitewait", or "needsdiagnosis" label.
+    This function gives all the "needs triage" issues.
```

Comment:
> This function gives all the "needs triage" issues.
	
I don't think that's quite right, this is the old method used to filter out the "new" issues.

Do we even need this method anymore? If there are no callers, it should be deleted.